tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -250,6 +251,17 @@ Verify "Enabled" Keeps Being Available After One Of The ISV Operators If Uninsta
    Uninstall Operator And Check Enabled Page Is Rendering  operator_name=${openvino_operator_name}  operator_appname=${openvino_appname}
    [Teardown]    Check And Uninstall Operator In Openshift    ${openvino_operator_name}   ${openvino_appname}
 
+Verify external links in Quick Starts are not broken
```

Comment:
this TC should go to `tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot`